airflow/providers/google/cloud/operators/dataproc_metastore.py

```diff
@@ -29,13 +29,139 @@
 from googleapiclient.errors import HttpError
 
 from airflow import AirflowException
-from airflow.models import BaseOperator
+from airflow.models import BaseOperator, BaseOperatorLink
+from airflow.models.taskinstance import TaskInstance
 from airflow.providers.google.cloud.hooks.dataproc_metastore import DataprocMetastoreHook
 
 if TYPE_CHECKING:
     from airflow.utils.context import Context
 
 
+BASE_LINK = "https://console.cloud.google.com"
+GCS_STORAGE_LINK = BASE_LINK + "/storage/browser/{destination_gcs_uri};tab=objects?project={project_id}"
+METASTORE_BASE_LINK = BASE_LINK + "/dataproc/metastore/services"
+METASTORE_BACKUP_LINK = METASTORE_BASE_LINK + "/{region}/{service_id}/backups/{backup_id}?project={project_id}"
```

Comment:
`/{region}/{service_id}/` you can move this to the `METASTORE_BASE_LINK` definition